akka-multi-node-testkit/src/main/scala/akka/remote/testkit/MultiNodeSpec.scala

```diff
@@ -171,18 +171,24 @@ object MultiNodeSpec {
   require(selfPort >= 0 && selfPort < 65535, "multinode.port is out of bounds: " + selfPort)
 
   /**
-   * UDP Port number to be used on this node. 0 means a random port.
+   * UDP Port number to be used when running Aeron tests. 0 means a random port.
+   *  (ie: AeronStreamConsistencySpec, AeronStreamLatencySpec, AeronStreamMaxThroughputSpec)
+   *
+   * Note: this port is used in a those Aeron tests, but not when running Akka Remote tests over UDP.
+   * Akka Remote tests using UDP will have the selfPort configured to point to a UDP port.
    *
    * {{{
-   * -Dmultinode.udp-port=0
+   * -Dmultinode.aeron-tests.port=0
    * }}}
    */
-  val udpPort: Option[Int] = Option(System.getProperty("multinode.udp-port")) match {
-    case None    => None
-    case Some(_) => Some(Integer.getInteger("multinode.udp-port", 0))
-  }
+  val aeronUdpPort: Option[Int] =
+    Option(System.getProperty("multinode.aeron-tests.port")).map { _ =>
```

Comment:
nitpick:
we renamed `udp-port` to `aeron-test.port` but in the code we are using it as `aeronUdpPort`, I personally find it more difficult to follow ...

Reply:
I agree, I will improve it. I think this property should moved out from this Spec class and be place on a more obvious way, maybe some trait used by the AeronStream* specs only.